src/pvr2wmc.h

```diff
@@ -1,25 +1,23 @@
-#pragma once
 /*
-*      Copyright (C) 2011 Pulse-Eight
-*      http://www.pulse-eight.com/
```

Comment:
Ive had a look at the pvr.demo code and I cant see anything that makes me think we need the pulse-eight copyright info

Of the 5 files that previously had it where ive now removed it:

src/client.h
header file, only contains declarations

src/client.cpp
boilerplate xbmc addon code that calls into pvr2wmc class

src/clientversion.h
Only contains the version string

src/pvr2wmc.h
Lars already confirmed this one is OK

src/pvr2wmc.cpp
I cant see anything problematic here, it really just sends backend calls and populates the XBMC structures